src/components/TodoModal/TodoModal.tsx

```diff
@@ -35,7 +35,29 @@ export const TodoModal: React.FC<Props> = React.memo(({
 
   const renderingModal = hasModalError
     ? (
-      <h3>Error occured when data loaded</h3>
+      <div className="modal-card">
+        <header
+          className="modal-card-head"
+          style={{ backgroundColor: '#ff6666' }}
+        >
+          <div
+            className="modal-card-title has-text-weight-medium"
+          >
+            ERROR
+          </div>
+
+          <button
+            aria-label="modal-close-button"
+            type="button"
+            className="delete"
+            onClick={() => onTodoSelected(null)}
+          />
+        </header>
+
+        <div className="modal-card-body">
+          <h3>Error occured when data loaded</h3>
+        </div>
+      </div>
```

Comment:
A lot of common code. Don't repeat yourself, use ternary, classNames or create a separate card component (f.e. you can pass the content of the bode using `children`). Optimize it.